src/test/java/org/sonatype/goodies/packageurl/PackageUrlParserTest.groovy

```diff
@@ -42,6 +42,13 @@ class PackageUrlParserTest
     return purl
   }
 
+  private PackageUrl parseAsIs(final String value) {
+    log "ParseAsIs: $value"
```

Comment:
maybe a dumb question: I'm not used to seeing logging in tests. Any chance this is leftover from some debugging?

Reply:
Just following @jdillon 's method above...

Reply:
@bradcupit log is a helper on goodies TestSupport